workshops/FCR/README.md

```diff
@@ -0,0 +1,44 @@
+# Fundamental of Teachable Machine
+Teachable Machine is a web-based tool that makes creating machine learning models fast, easy, and accessible to everyone.Educators, artists, students, innovators, makers of all kinds – really, anyone who has an idea they want to explore. No prerequisite machine learning knowledge required.You train a computer to recognize your images, sounds, and poses without writing any machine learning code. Then, use your model in your own projects, sites, apps, and more.
```

Comment:
Teachable Machine is a web-based tool that makes creating machine learning models fast, easy, and accessible to everyone. Educators, artists, students, innovators, makers of all kinds – really, anyone who has an idea they want to explore. No prerequisite machine learning knowledge is required. You train a computer to recognize your images, sounds and poses without writing any machine learning code. Then, use your model in your projects, sites, apps, and more.